src/Voron/Impl/FileHeaders/MetadataFile.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Voron.Impl.FileHeaders;
+
+[StructLayout(LayoutKind.Explicit, Pack = 1)]
+public struct MetadataFile
```

Comment:
Better structure would be:

`ulong Hash` as the first value - computed over the entire rest of the file
`int Version` - which also tells us what is the size.

Then the rest of the fields.

Reply:
done.
no need to size by version will simply rely on the file size